site/docs/pages/temp-demo.mdx

```diff
@@ -0,0 +1,7 @@
+
+<div className="flex justify-center p-4">
+  <div className="text-center">
+    <h1 className="text-3xl md:text-4xl">Temp Demo Page</h1>
+    <p className="text-lg md:text-xl">Temporary demo page to test geo blocking restrictions.</p>
+  </div>
+</div>
```

Comment:
Temporary - to be deleted once we've verified the geo blocking feature in production.